stdpopsim/cli.py

```diff
@@ -329,6 +329,9 @@ def add_simulate_species_parser(parser, species):
     species_parser.add_argument(
         "-q", "--quiet", action='store_true',
         help="Do not write out citation information")
+    species_parser.add_argument(
+        "-d", "--dry-run", action='store_true', default=False,
+        help="Print out summary of simulation configurations without running simulation")
```

Comment:
More accurate would be "Do not run the actual simulation". It's possible to have ``--dry-run`` ``--quiet`` and print out nothing (although, I'm not sure how helpful that is)